web/src/pages/Info/List.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { PageContainer } from '@ant-design/pro-layout';
+import { useIntl } from 'umi';
+
+const Info: React.FC = () => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <PageContainer title={formatMessage({ id: 'page.info.pageContainer.title' })}>
+      <select className="selector" id="selector" style={{ backgroundColor: "#fff", width: "100%", height: "24px", textAlign: "right", padding: "16px 24px", marginBottom: "15px" }}>
+        <option value="categoryA">CategoryA</option>
+        <option value="categoryB">CategoryB</option>
+        <option value="categoryC">CategoryC</option>
+      </select>
+      <div className="wrap" style={{ width: "100%", height: "740px", padding: "16px 24px", margin: "0 auto", backgroundColor: "#fff" }}>
```

Comment:
Using div and other HTML tags look good to me, but you should move those inline style to a standalone file.